test/kpca.jl

```diff
@@ -71,8 +71,10 @@ K = MultivariateStats.pairwise((x,y)->x'*y, X)
 @test_throws AssertionError fit(KernelPCA, rand(1,10), kernel=nothing) # symmetric kernel
 M = fit(KernelPCA, K, maxoutdim = 5, kernel=nothing, inverse=true) # use precomputed kernel
 M2 = fit(PCA, X, method=:cov, pratio=1.0)
-@test_throws AssertionError reconstruct(M, X) # no reconstruction for precomputed kernel
-@test abs.(transform(M)) ≈ abs.(transform(M2, X))
+@test_throws ArgumentError reconstruct(M, X) # no reconstruction for precomputed kernel
+@test abs.(transform(M)) ≈ abs.(transform(M2, X)) # Base.Test.test_approx_eq_modphase
```

Comment:
Why mention `Base.Test.test_approx_eq_modphase` in the comment? Either remove it or make it more explicit.